app/model/family.py

```diff
@@ -23,7 +23,7 @@ class FamilyReadDTO(BaseModel):
     documents: list[str]
     collections: list[str]
     organisation: str
-    corpus_id: str
+    corpus_import_id: str
```

Comment:
Good shout - I did call this `corpus_id` in teh nav frontend as it doesn't use `import_id` for anything

Reply:
Nice! I contemplated but I think this is more consistent with how we're doing it in this repo, so thought I'd change for now!